docs/release/trg-0/trg-4-02.md

```diff
@@ -0,0 +1,156 @@
+publishing them on DockerHub. In case you are using Docker images for build or testing purposes (for example _pandoc_ or
+_cypress_, etc.) and you do not publish the images, you can use other publicly available image, as long as the tools are
+open source license compliant.
+
+For automated TRG checks, you can skip base image checks on Dockerfiles by declaring it in the `.tractusx` metadata
+files. Details can be found in
+the [metadata file documentation](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md)
+:::
+
+### Use minor or major Image Tag
+
+Do not use an image tag pointing to a patch version of the image. Instead, use an image tag containing either minor or
+major version.
+
+To keep application images compliant, up-to-date and secure it is mandatory to use image tags which allows to rebuild
+the applications container image without touching the `DOCKERFILE` to pull an updated version of the base image (e.g. to
+fix a potential problem inside the base image.
+
+:::warning Do not use image tag with patch version
+
+```DOCKERFILE
+FROM alpine:3.18.5 as BUILDER
+
+COPY . /app/build
+WORKDIR /app/build
+RUN build
+
+FROM alpine:3.18.5
+
+COPY --from=BUILDER /app/build /app
+[...]
+```
+
+:::
+
+:::tip Use image tag with minor or major version
+
+```DOCKERFILE
+FROM alpine:3 as BUILDER
+
+COPY . /app/build
+WORKDIR /app/build
+RUN build
+
+FROM alpine:3.18
+
+COPY --from=BUILDER /app/build /app
+
+[...]
+```
+
+:::
+
+The next time the applications container image is created, an updated base image will be used, if available.
+
+### Use Base Image as-is
+
+It is recommended to use the base image **as-is** and to not modify or update it.
+
+Do not use package manager update mechanisms to update the runtime of the base image (e.g. to fix security
+vulnerabilities). Updating or modifying the base image might pull in a new version of a library which is no longer under
+the correct Intellectual Property and/or license.
+
+Same applies for installing additional tools, libraries, commands, which modifies the base image. If possible, select a
+base image for your application that contains all dependent tools.
+
+:::warning Do not update or install additional packages
+
+```DOCKERFILE
+# Call for feedback: Does this example make sense at all or 
+# should it be removed?
+FROM alpine:3.18.2
+
+COPY some/binary /app/
+RUN apk update && apk upgrade \ # fix CVE-2023-123
+    apk add someTool            # install pkg "someTool"
+```
+
+:::
+
+In the example a patch level image tag of alpine is used, which is not recommended, and an update/upgrade is done to get
+the latest versions of all installed packages installed. Instead, use a minor image tag (e.g. `alpine:3.18`) as
+recommended in [_Use minor or major Image Tag_](#use-minor-or-major-image-tag).
+
+An exception from modifying the base image might be a build stage in the `DOCKERFILE`, as long as the resulting image
```

Comment:
Would again write it like it is usually used: `Dockerfile`

Reply:
Fixed with latest commit.